### ISSUE TITLE
DemoBench: NPE when typing "help" at CRaSH command line.

### ISSUE THREAD

**Issue by chrisr3** [corda/corda#437]
Install DemoBench from DMG (MacOSX) or RPM (Fedora/Linux). Launch a Corda node and then type `help` at the CRaSH command line:
```
Mon Mar 27 15:59:34 BST 2017>>> help
help: exception: NullPointerException
```
This problem does _not_ happen when running DemoBench as follows:
```
$ gradlew :tools:demobench:installDist
$ cd tools/demobench/build/install/demobench
$ bin/demobench
```

**Comment by chrisr3** [comment]
My initial guess is that `javapackager` has not included some part of the JRE that the `help` function is relying on.

**Comment by chrisr3** [comment]
The missing piece appears to be `tools.jar`, and adding that back fixes the NPE on Fedora/Linux. But it's not that simple on MacOSX.

**Comment by chrisr3** [comment]
@mikehearn 
> tools.jar ... CRaSH assumes it can compile java code at runtime.
> which is silly of course. there is a JIRA task to fix that. it's not hard to do, i might take a crack at it tomorrow.

**Comment by chrisr3** [comment]
On MacOSX, `tools.jar` must be copied to:
```
DemoBench.app/Contents/PlugIns/Java.runtime/Contents/Home/lib/tools.jar
```

**Comment by chrisr3** [comment]
Fixed by #461.
